src/bin/nydus-image/inspect.rs

```diff
@@ -272,8 +305,51 @@ Compressed Size:    {compressed_size}
         Ok(None)
     }
 
+    // Convert an inode number to a file path, the rafs v6 file is handled separately.
+    fn path_from_ino(&mut self, ino: u64) -> Result<Vec<PathBuf>, anyhow::Error> {
```

Comment:
Better to comment that it can find all hardlinks sharing the same inode number

Reply:
Done.